app/services/sweeper.py

```
@@ -0,0 +1,239 @@
+import asyncio
+import logging
+import random
+import time
+from datetime import datetime, timedelta, timezone
+from typing import Callable
```

Ruff (0.14.10)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)